news/fix_is_current.rst

```diff
@@ -0,0 +1,13 @@
+**Added:**
+    * New date functions to dates.py
```

Comment:
please be more specific.  What new functionality are you giving the people

Reply:
I can change the title and it passes all tests.  I am not sure what codecov/project is testing for because all the new functions work and they build correctly.